test/service/data_generator/fake_data_utils.py

```diff
@@ -69,13 +70,13 @@ def load_data(self):
                     fake_data_body += json.dumps({"index": {"_type": "meta", "_id": i}}) + "\n"
                     fake_data_body += json.dumps(faker.generate_fake(self.doc_template)) + "\n"
                 self.elasticsearch_client.bulk(fake_data_body, index=index, doc_type='meta', refresh='wait_for')
-        except elasticsearch5.exceptions.NotFoundError:
+        except ESExceptions.NotFoundError:
```

Comment:
It depends on how we import the module. I usually try to be more specific about what the code needs from a particular module. For example, if I need only very few classes, variables or functions, I will just go with something like `from elasticsearch.exceptions import NotFoundError` and thus `except NotFoundError`. Otherwise, I go with `import elasticsearch`. As PEP8 does not specify anything regarding to this, do we have a written coding convention or guideline on writing `except`?